Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Notifications/UX/Toast/KryptonToastNotificationVersion2.cs

```diff
@@ -165,6 +165,16 @@ private void InitializeComponent()
             this.ResumeLayout(false);
 
         }
+
+        protected override void Dispose(bool disposing)
```

Comment:
What use is this ?

Reply:
See my previous comment. Will be redoing the notifications module, as there are a # of bugs (see Discord chat for more info).